docfx.json

```diff
@@ -0,0 +1,43 @@
+{
+  "metadata": [
+    {
+      "src": [
+        {
+          "files": [
+            "src/**.csproj"
+          ],
+          "exclude": [
+            "**/bin/**",
```

Comment:
Are these needed?

Reply:
The entire 'metadata' section in docfx.json can be removed since it is never called. I've been meaning to do that. Would you prefer it be removed?

Reply:
Yes. This section is intended to generate API reference docs for OpenTelemetry .NET, it has no value to OpenTelemetry C++.